radio/util/crossfire-parse.py

```diff
@@ -70,28 +71,47 @@ def dump(data, maxLen=None):
     0xAD, 0x78, 0xD2, 0x07, 0x53, 0x86, 0x2C, 0xF9
 ]
 
+# logs data are BigEndian
+def getCrossfireValue(signed, data=[], *args):
+    hexstring = "0x"
+    for n in data:
+        hexstring += hex(n)[2:]
+    value = BitArray(hex=hexstring)
+    if (signed is True):
+        return value.int
+    else:
+        return value.uint
+
 def crc8(buffer):
     crc = 0
     for c in buffer:
         crc = crc8tab[crc ^ c]
     return crc
 
+# note : python arrays slicing is a bit strange payload[x:y] means from 'x' included to 'y' NOT included
+#        therefore y - x gives the number of bytes sliced.
 def ParseGPS(payload):
-    pass
+    lat = getCrossfireValue(True, payload[0:4]) / 1e7
```

Comment:
do you mean 0x1e7?

Reply:
Not quite, I mean 10 000 000

Python 2.7.13 (default, Jan 19 2017, 14:48:08) 
[GCC 6.3.0 20170118] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> print 1e7
10000000.0
>>>

Reply:
Much easier to read than counting zeros, IMHO.  And another reason hex should be written with uppercase chars...  :)